src/views/List.jsx

```diff
@@ -20,52 +20,58 @@ export function List({ data, listPath }) {
 	});
 
 	function handleInputChange(e) {
+		e.preventDefault();
 		setInput(e.target.value);
 	}
 
 	function clearSearch() {
 		setInput('');
 	}
 
+	const listName = listPath?.substring(listPath.indexOf('/') + 1);
+
+	if (data.data.length === 0) {
+		return (
+			<>
+				<h2>{listName}</h2>
+				<AddItemForm listPath={listPath} />
+			</>
+		);
+	}
+
 	if (data.loading) {
 		// If data is not loaded yet, render a loading indicator
 		return <p>Loading...</p>;
 	}
 
 	return (
 		<>
-			<form action="">
-				<label htmlFor="searchItems">Search Items: </label>
-				<input
-					onChange={handleInputChange}
-					id="searchItems"
-					value={input}
-					type="text"
-				/>
-			</form>
-			<button onClick={clearSearch}>X</button>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
+			<h2>{listName}</h2>
```

Comment:
There are some items here that look like it could be moved into a new component, something that is super nice in React is having very readable JSX. So take a look at the JSX you have currently and see how some of these items could be moved into components. For example below `AddItemForm` is very clear on what it does and if I want to read what it does I can click into it.